### PR TITLE
chore: added eventListenerHook

### PR DESCRIPTION
**Pull request by anikethsaha**
Followup of https://github.com/itaditya/react-hook-recipes/pull/3

### CODE REVIEW

**Comment by itaditya** [comment]
@anikethsaha can you pull the latest changes from master and update your PR?

**Comment by anikethsaha** [comment]
@itaditya Please take a look if this is fine !

File: hooks.js [hunk 1]
```diff
@@ -220,6 +220,51 @@ export const hooks = [
         </h1>
       );
     }
+=======
```
Comment (itaditya) [inline hooks.js]: remove this line

File: hooks.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export const hooks = [
   {
     name: "useArray",
-    author: 'kitze',
+    author: "kitze",
```
Comment (itaditya) [inline hooks.js]: keep it as single quotes only

File: hooks.js [hunk 3]
```diff
@@ -151,6 +154,73 @@ export const hooks = [
     );
   }
     `
+  },
+  {
+    name: "useEventListener",
+    author: "Gabe Ragland",
+    link: "https://codesandbox.io/s/z64on3ypm",
+    description:
+      "react hook that checks if addEventListener is supported, add the event listener, and removal it on cleanup",
```
Comment (itaditya) [inline hooks.js]:
```suggestion
      "react hook that checks if addEventListener is supported, add the event listener, and remove it during cleanup",
```

File: hooks.js (hunk 1 above)
Comment (anikethsaha) [reply hooks.js]: done 👍

File: hooks.js (hunk 2 above)
Comment (anikethsaha) [reply hooks.js]: These got changed during `formatOnSave` with vscode.
will change it but do you have plan of adding linters ?

**Comment by itaditya** [comment]
Congrats on getting the PR merged

Congrats

**Pull request merged**